d2l-button.html

```diff
@@ -91,7 +91,10 @@
 	<script>
 		Polymer({
 			is: 'd2l-button',
-			behaviors: [window.D2L.PolymerBehaviors.Button.Behavior]
+			behaviors: [window.D2L.PolymerBehaviors.Button.Behavior],
+			focus: function() {
```

Comment:
I put this implementation here (as opposed to the behavior), since the button element is specific to this component.  Alternatively, I _could_ make the behavior assume it will find a `button` element, or have it call a helper method on the component to get the element if we really want to move this to the behavior.